Sources/SwiftSyntaxMacroExpansion/MacroExpansion.swift

```diff
@@ -236,6 +236,7 @@ public func expandAttachedMacroWithoutCollapsing<Context: MacroExpansionContext>
       let members = try attachedMacro.expansion(
         of: attributeNode,
         providingMembersOf: declGroup,
+        missingConformancesTo: conformanceList?.map(\.type) ?? [],
```

Comment:
Why is this parameter called `missingConformancesTo:`? AFAICT you are passing in the existing conformances, not some list of missing conformances.

Reply:
I'm struggling mightily to name this. Let's discussion naming over at https://forums.swift.org/t/pitch-member-macros-that-know-what-conformances-are-missing/66590